codama-nodes/src/instruction_remaining_accounts_node.rs

```diff
@@ -5,9 +5,13 @@ use codama_nodes_derive::{node, node_union};
 pub struct InstructionRemainingAccountsNode {
     // Data.
     #[serde(default)]
-    #[serde(skip_serializing_if = "std::ops::Not::not")]
+    #[serde(skip_serializing_if = "crate::is_default")]
     pub is_optional: bool,
+    #[serde(default)]
+    #[serde(skip_serializing_if = "crate::is_default")]
     pub is_signer: IsAccountSigner,
+    #[serde(default)]
+    #[serde(skip_serializing_if = "crate::is_default")]
     pub is_writable: bool,
     #[serde(default)]
     #[serde(skip_serializing_if = "Docs::is_empty")]
```

Comment:
Nit: are we able to use `crate::is_default` here as well?

Reply:
Yeah I think it'd be great to have that new `crate::is_default` helper everywhere. :)

Reply:
Done!